src/Lykke.Service.Assets/Cache/AssetsForClientCacheManager.cs

```diff
@@ -32,144 +37,109 @@ public class AssetsForClientCacheManager : IAssetsForClientCacheManager
             _log = log;
         }
 
-        public async Task ClearCache(string reason)
+        public async Task ClearCacheAsync(string reason)
         {
-            var count = 0;
-            while(true)
-            {
-                var keys = _redisServer.Keys(pattern: _settings.InstanceName + ":Assets:Client:*", pageSize: 1000).ToArray();
-                count += keys.Length;
-                if (!keys.Any()) break;
-                await _redisDatabase.KeyDeleteAsync(keys);
-            }
+            RedisKey[] keys = _redisServer.Keys(pattern: $"{_settings.InstanceName}{PatternClient}*", pageSize: 1000).ToArray();
 
-            await _log.WriteInfoAsync(nameof(AssetsForClientCacheManager), nameof(ClearCache), $"Clear assets cache, count of record: {count}, reason: {reason}");
+            await _redisDatabase.KeyDeleteAsync(keys);
+            
+            await _log.WriteInfoAsync(nameof(AssetsForClientCacheManager), nameof(ClearCacheAsync), $"Clear assets cache, count of record: {keys.Length}, reason: {reason}");
         }
 
-        public async Task RemoveClientFromChache(string clientId)
+        public async Task RemoveClientFromCacheAsync(string clientId)
         {
             try
             {
-                await _cache.RemoveAsync(GetKeyAvailableAssets(clientId, true));
-                await _cache.RemoveAsync(GetKeyAvailableAssets(clientId, false));
-                await _cache.RemoveAsync(GetKeyCashInViaBankCardEnabled(clientId, true));
-                await _cache.RemoveAsync(GetKeyCashInViaBankCardEnabled(clientId, false));
-                await _cache.RemoveAsync(GetKeySwiftDepositEnabled(clientId, true));
-                await _cache.RemoveAsync(GetKeySwiftDepositEnabled(clientId, false));
+                await Task.WhenAll(
+                    _cache.RemoveAsync(GetKeyAvailableAssets(clientId, true)),
+                    _cache.RemoveAsync(GetKeyAvailableAssets(clientId, false)),
+                    _cache.RemoveAsync(GetKeyCashInViaBankCardEnabled(clientId, true)),
+                    _cache.RemoveAsync(GetKeyCashInViaBankCardEnabled(clientId, false)),
+                    _cache.RemoveAsync(GetKeySwiftDepositEnabled(clientId, true)),
+                    _cache.RemoveAsync(GetKeySwiftDepositEnabled(clientId, false)),
+                    _cache.RemoveAsync(GetKeyAssetConditions(clientId)));
             }
-            catch (Exception ex)
+            catch (Exception exception)
             {
-                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(RemoveClientFromChache), clientId, ex);
+                await _log.WriteErrorAsync(nameof(AssetsForClientCacheManager), nameof(RemoveClientFromCacheAsync), clientId, exception);
```

Comment:
Is this error for the developer or DevOps? If for the developer then should be warning

Reply:
Yes, this is an error for DevOps it means something wrong with cache.